.packit.yaml

```diff
@@ -239,38 +301,110 @@ jobs:
     LEAPPDATA_BRANCH: "upstream"
     LEAPP_DEVEL_TARGET_RELEASE: "9.2"
 
-# - job: tests
-#   fmf_url: "https://gitlab.cee.redhat.com/oamg/leapp-tests"
-#   fmf_ref: "master"
-#   use_internal_tf: True
-#   trigger: pull_request
-#   targets:
-#     epel-8-x86_64:
-#       distros: [RHEL-8.6.0-Nightly]
-#   identifier: tests-8.6to9.0-sst
-#   tmt_plan: "^(?!.*tier[2-3].*)(.*max_sst.*)"
-#   tf_post_install_script: "#!/bin/sh\nsudo sed -i s/.*ssh-rsa/ssh-rsa/ /root/.ssh/authorized_keys"
-#   tf_extra_params:
-#     environments:
-#       - tmt:
-#           context:
-#             distro: "rhel-8.6"
-#   env:
-#     SOURCE_RELEASE: "8.6"
-#     TARGET_RELEASE: "9.0"
-#     RHSM_REPOS: "rhel-8-for-x86_64-appstream-eus-rpms,rhel-8-for-x86_64-baseos-eus-rpms"
-#     LEAPPDATA_BRANCH: "upstream"
+# On-demand minimal beaker tests
+- &beaker-minimal-88to92
+  <<: *beaker-minimal-86to90
+  labels:
+    - beaker-minimal
+    - beaker-minimal-8.8to9.2
+  targets:
+    epel-8-x86_64:
+      distros: [RHEL-8.8.0-Nightly]
+  identifier: tests-8.8to9.2-beaker-minimal
+  tf_extra_params:
+    environments:
+      - tmt:
+          context:
+            distro: "rhel-8.8"
+        # tag resources as sst_upgrades to enable cost metrics collection
+        settings:
+          provisioning:
+            post_install_script: "#!/bin/sh\nsudo sed -i s/.*ssh-rsa/ssh-rsa/ /root/.ssh/authorized_keys"
+            tags:
+              BusinessUnit: sst_upgrades
+  env:
+    SOURCE_RELEASE: "8.8"
+    TARGET_RELEASE: "9.2"
+    RHSM_REPOS: "rhel-8-for-x86_64-appstream-beta-rpms,rhel-8-for-x86_64-baseos-beta-rpms"
```

Comment:
The repos are not beta anymore. TLDR about how to specify RHSM_REPOS is:

- If repos are not eus or beta, don't specify any RHSM_REPOS, there's a configuration mapping in the testing project that would do it automatically. This prevents us from changing configuration values in two different places.
- If repos are eus, as is the case in 86to90, define this envvar `RHSM_REPOS_EUS: "eus"` and this should resolve correctly in the configuration mapping.

Reply:
What should I put there?

Reply:
In case of 88to92, there's no need to specify `RHSM_REPOS` as it's not beta anymore. The value will be taken from the configuration mapping in the tests project.

Reply:
Removed RHSM_REPOS for 8.8->9.2, kept only for 8.9->9.3; Changed RHSM_REPOS_EUS for all 8.6->9.0 tests.